addons/account/views/account_move_views.xml

```diff
@@ -1703,7 +1703,7 @@
             <field name="view_mode">tree</field>
             <field name="view_id" ref="view_move_line_payment_tree"/>
             <field name="search_view_id" ref="view_account_move_line_payment_filter"/>
-            <field name="domain">[('parent_state', '=', 'posted'), ('date_maturity', '!=', False), ('amount_residual', '&gt;', 0), ('account_id.reconcile', '=', True), ('journal_id.type', '=', 'sale')]</field>
+            <field name="domain">[('parent_state', '=', 'posted'), ('date_maturity', '!=', False), ('move_id.payment_state', 'in', ('not_paid', 'partial')), ('amount_residual', '&gt;', 0), ('account_id.reconcile', '=', True),  ('account_id.account_type', '=', 'asset_receivable'), ('display_type', '=', 'payment_term'), ('journal_id.type', '=', 'sale')]</field>
```

Comment:
The condition on the display_type seems a bit useless as well ; isn't adding the account_type enough ?